apps/ledger-live-desktop/src/renderer/families/stellar/AddAssetModal/fields/AssetSelector.tsx

```diff
@@ -106,16 +106,44 @@ export default function DelegationSelectorField({
   onChange: (_: TokenCurrency | undefined | null) => void;
 }) {
   const [query, setQuery] = useState("");
+  const [lastItemIndex, setLastItemIndex] = useState<number | undefined>(undefined);
+  const [keepLastScrollPosition, setLastKeepScrollPosition] = useState(false);
+
   const subAccounts = account.subAccounts;
-  const options = listTokensForCryptoCurrency(account.currency);
+
+  const { data, isLoading, loadNext } = useTokensData({
+    networkFamily: [account.currency.id],
+    pageSize: 100,
+  });
+
+  const options = useMemo(() => {
+    if (!isLoading) {
+      return data?.tokens;
+    }
+    return [];
+  }, [data?.tokens, isLoading]);
```

Comment:
```suggestion
  const options = useMemo(() =>  data?.tokens  || [], [data?.tokens]);
```

I think you don't need `isLoading`